C:\Users\user\Pictures\Moughni\Lewo>npm run dev

> lewo-server@1.0.0 dev
> nodemon server.js

[nodemon] 3.1.10
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: js,mjs,cjs,json
[nodemon] starting `node server.js`
Ignoring invalid configuration option passed to Connection: acquireTimeout. This is currently a warning, but in future versions of MySQL2, an error will be thrown if you pass an invalid configuration option to a Connection
Ignoring invalid configuration option passed to Connection: timeout. This is currently a warning, but in future versions of MySQL2, an error will be thrown if you pass an invalid configuration option to a Connection
Ignoring invalid configuration option passed to Connection: reconnect. This is currently a warning, but in future versions of MySQL2, an error will be thrown if you pass an invalid configuration option to a Connection
C:\Users\user\Pictures\Moughni\Lewo\node_modules\express\lib\router\route.js:216
        throw new Error(msg);
        ^

Error: Route.get() requires a callback function but got a [object Object]
    at Route.<computed> [as get] (C:\Users\user\Pictures\Moughni\Lewo\node_modules\express\lib\router\route.js:216:15)
    at proto.<computed> [as get] (C:\Users\user\Pictures\Moughni\Lewo\node_modules\express\lib\router\index.js:521:19)
    at Object.<anonymous> (C:\Users\user\Pictures\Moughni\Lewo\routes\mentorships.js:7:8)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object..js (node:internal/modules/cjs/loader:1706:10)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)

Node.js v22.14.0
[nodemon] app crashed - waiting for file changes before starting...
